apps/web/src/pages/templates/components/LackIntegrationAlert.tsx

```diff
@@ -1,29 +1,30 @@
 import { useState } from 'react';
 import styled from '@emotion/styled';
+
 import { ChannelTypeEnum } from '@novu/shared';
 
 import { Text } from '../../../design-system';
-import { CircleArrowRight } from '../../../design-system/icons/arrows/CircleArrowRight';
+import { CircleArrowRight } from '../../../design-system/icons';
 import { IntegrationsStoreModal } from '../../integrations/IntegrationsStoreModal';
 import { useSegment } from '../../../components/providers/SegmentProvider';
 import { stepNames, TemplateEditorAnalyticsEnum } from '../constants';
 import { useIsMultiProviderConfigurationEnabled } from '../../../hooks';
 import { IntegrationsListModal } from '../../integrations/IntegrationsListModal';
 
-const DoubleArrowRightStyled = styled(CircleArrowRight)`
-  cursor: pointer;
-`;
+type alertType = 'error' | 'warning';
 
-export function LackIntegrationError({
+export function LackIntegrationAlert({
   channelType,
   text,
+  type = 'error',
```

Comment:
Extended the component to switch between warning and error types, maybe in the future we will have success as well.